src/main/java/org/jabref/gui/mergeentries/MultiMergeEntriesView.java

```diff
@@ -232,64 +233,67 @@ private boolean isMultilineField(Field field) {
         public Cell(String content, Field field, int columnIndex) {
             this.content = content;
 
-            FieldRow row = fieldRows.get(field);
-
-            prefWidthProperty().bind(((Region) supplierHeader.getChildren().get(columnIndex)).widthProperty());
-            setMinWidth(Control.USE_PREF_SIZE);
-            setMaxWidth(Control.USE_PREF_SIZE);
-            prefHeightProperty().bind(((Region) fieldEditor.getChildren().get(row.rowIndex)).heightProperty());
-            setMinHeight(Control.USE_PREF_SIZE);
-            setMaxHeight(Control.USE_PREF_SIZE);
-
-            // Button
-            ToggleButton cellButton = new ToggleButton();
-            cellButton.prefHeightProperty().bind(heightProperty());
-            cellButton.setMinHeight(Control.USE_PREF_SIZE);
-            cellButton.setMaxHeight(Control.USE_PREF_SIZE);
-            getChildren().add(cellButton);
-            HBox.setHgrow(cellButton, Priority.ALWAYS);
-
-            // Text
-            DiffHighlightingEllipsingTextFlow buttonText = new DiffHighlightingEllipsingTextFlow(content);
-
-            buttonText.maxWidthProperty().bind(widthProperty());
-            buttonText.maxHeightProperty().bind(heightProperty());
-            cellButton.setGraphic(buttonText);
-
-            // Tooltip
-            Tooltip buttonTooltip = new Tooltip(content);
-            buttonTooltip.setWrapText(true);
-            buttonTooltip.prefWidthProperty().bind(widthProperty());
-            buttonTooltip.setTextAlignment(TextAlignment.LEFT);
-            cellButton.setTooltip(buttonTooltip);
-
-            cellButton.setToggleGroup(row.toggleGroup);
-            if (row.toggleGroup.getSelectedToggle() == null) {
-                cellButton.setSelected(true);
-            }
+            DefaultTaskExecutor.runInJavaFXThread(() -> {
```

Comment:
I would suggest adding a  short comment on why this is necessary here at this point

Reply:
I'm still not entirely sure why it is necessary. Would giving the effect that we avoid be enough?

(Also, I'm sorry for not mentioning you as co-author of 4c74d51)